### PR TITLE
added Mockito test

### PR DESCRIPTION
**Pull request by sarayaffe**

### CODE REVIEW

File: src/main/java/ScrabbleGame.java [hunk 1]
```diff
@@ -54,14 +58,17 @@ private boolean isWordInTiles(char[] wordLetters) {
...
+            if (!tiles.remove((Character) letter)) {
                 tiles = tilesCopy;
                 return false;
             }
+//            (tiles.contains(letter)) {
+//                tiles.remove(Character.valueOf(letter));
+//            } else {
+//                tiles = tilesCopy;
+//                return false;
+//            }
```
Comment (andrewoid) [inline src/main/java/ScrabbleGame.java]: Don't keep commented out code.

File: src/main/java/ScrabbleGame.java [hunk 2]
```diff
@@ -2,26 +2,30 @@
...
         }
     }
 
+
     public void playGame(Scanner scanner) {
         while (true) {
             System.out.println("Enter word with given letters or 'exit' to end game\n" + tiles);
-            String word = scanner.nextLine().toLowerCase();
+            String word = scanner.nextLine().toUpperCase();
             if (word.equalsIgnoreCase("exit")) {
```
Comment (andrewoid) [inline src/main/java/ScrabbleGame.java]: If you convert the line to upper case, you no longer need to do equalsIgnoreCase()

File: src/main/java/ScrabbleGame.java [hunk 3]
```diff
@@ -2,27 +2,31 @@
...
 
     private void initializeTiles() {
         for (int i = 0; i < 7; i++) {
-            tiles.add((char) (rnd.nextInt(26) + 'a'));
+            tiles.add(letterPool.getRandomLetter());
         }
     }
 
+
     public void playGame(Scanner scanner) {
```
Comment (andrewoid) [inline src/main/java/ScrabbleGame.java]: You should test this method too, since you wrote it. Can you figure out how?

Comment (sarayaffe) [reply src/main/java/ScrabbleGame.java]: I am not really sure how to test playGame() with the while loop that's included there, so I modified it, but now it seems to be an unnecessary method to include at all.

**Pull request merged**